libraries/AP_Airspeed/AP_Airspeed.cpp

```diff
@@ -294,6 +294,36 @@ AP_Airspeed::AP_Airspeed()
     _singleton = this;
 }
 
+// macro for use by HAL_INS_PROBE_LIST
+#define GET_I2C_DEVICE(bus, address) hal.i2c_mgr->get_device(bus, address)
+
+bool AP_Airspeed::add_backend(AP_Airspeed_Backend *backend)
+{
+    if (!backend) {
+        return false;
+    }
+    if (num_sensors >= AIRSPEED_MAX_SENSORS) {
+        AP_HAL::panic("Too many airspeed drivers");
```

Comment:
This is great and all, but using the `ADD_BACKEND` macro means you will almost never see this message, and instead we will just not create the sensor, and won't emit any form of warning.

Reply:
which is the same as compass, baro and imu
a message on a peripheral isn't all that useful